src/parser.ts

```diff
@@ -78,8 +81,8 @@ export class Parser {
         }
     }
 
-    latexmkSkipped(log: string) : boolean {
-        const lines = log.replace(/(\r\n)|\r/g, '\n').split('\n')
+    latexmkSkipped(log: string): boolean {
+        const lines = log.split('\n')
```

Comment:
I do not know, but that doesn't matter much, given the adjustments I made in [line 50](https://github.com/James-Yu/LaTeX-Workshop/pull/205/files/b1db393f45414370fcce405fa364d29d20f90d86#diff-6518fffc4e430fcc63cd30eb07131a95R50), where the repeated replaces were centralized